README.md

```diff
@@ -1,56 +1,42 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Tour Booking App (backend)
```

Comment:
- [ ] You are using Rspec, but the "test" folder still exists, which is used by another test framework.  Kindly delete the folder if you are not using it.